config/etcd.yaml

```diff
@@ -2,7 +2,6 @@ replicaCount: 3
 persistence:
   enabled: true
   size: 10Gi
-  storageClass: "premium-rwo"
```

Comment:
nit: Do you think it'd make sense to create a storage class for AKS as well (like for AWS) and use it?The default one has very limited iops, so etcd may struggle.
Example one here:
https://github.com/bufbuild/core/blob/a96ec9738ac0e3c0158ed6d2027b87f396bebc67/data/k8s/azure/aberdeen/aks_bufbuild-aberdeen-1_bufbuild-aberdeen-usc1_aberdeen-usc1/argocd/premium2-storage-class.yaml#L1-L15

Reply:
I'm ok with it, but we're not recommeding it in https://buf.build/docs/bufstream/deployment/azure/deploy-etcd/.  I think we should try to keep these in sync?

Reply:
We are recommending it there:
<img width="829" alt="image" src="https://github.com/user-attachments/assets/bfb32d13-34c8-49a5-92b1-38a2cd3b5293" />
 😅